src/Hacks/antiaim.cpp

```diff
@@ -590,25 +604,7 @@ void DoAntiAimY(QAngle& angle, int command_number, bool bFlip, bool& clamp)
 		default:
 			angle.y -= 0.0f;
 			break;
-		case AntiAimType_Y::ANGEL_SPIN:
-			clamp = false;
-			factor = (globalVars->curtime * 5000.0f);
-			angle.y = factor + 36000000.0f;
-			break;
-		case AntiAimType_Y::LUA_UNCLAMPED:
-			clamp = false;
-			angle.y = LuaScriptY( lastAngleY, angle.y );
-			break;
-		case AntiAimType_Y::LUA_UNCLAMPED2:
-			clamp = false;
-			angle.y = LuaScriptY2( lastAngleY2, angle.y );
-			lastAngleY2 = angle.y;
-			break;
-		default:
-			angle.y -= 0.0f;
-			break;
 	}
-	lastAngleY = angle.y;
```

Comment:
This line is needed for Lua Scripts.

However I'll go ahead and merge then fix it.